packages/components/src/internal/components/editable/models.test.ts

```diff
@@ -246,6 +249,43 @@ describe('EditorModel', () => {
             expect(errors).toHaveLength(1);
         });
 
+        test('missing required for generated unique id', () => {
```

Comment:
I can't quite parse this test name. It's testing that we don't mark the generated Id columns as required, right? How about something like "generated unique id data not required"?

Reply:
done